### ISSUE TITLE
as an admin I need to complete several actions from the index view

### ISSUE THREAD

**Issue by jordanskole** [autorivalry/ar-admin#83]
As user, we need a way to display some additional information and multiple actions from the index view. 

We need to display the hero vs. vehicle under the title, and need actions to show the edit view in addition to just the grab code (what we have now)

Read more - https://material.angularjs.org/latest/demo/menu
Waiting on - https://github.com/angular/material/issues/2595#issuecomment-167953291

**Comment by jordanskole** [comment]
added the menu items, but I dont like the way they look/work. It will do for now, but I am going to create another issue that adds additional action items. 

Also, I think there should be a single detail view (see issue #75) and include 3 tabs on that view.
